File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>
#include <stdlib.h>
#include <string.h>
int main(void)
{   
    int height;
    // It promts the user for a number between 1 and 8
    do
    {
        height = get_int("Height: ");
    
    }
    while (height < 1 || height > 8);
    
    //numS is the number of spaces to be printed
    int numS = height - 1;
    int row = 1;
    char spaces[height];
    char hashes[height];
    strcpy(spaces, "");
    
    do
    {
        //This is a function that builds all the spaces
        for (int i = 0; i < numS; i++)
        {
            strcat(spaces, " ");
        }
        //This line builds up my hashes
        strcat(hashes, "#");
        //print a line of spaces, hashes and hashes to form my pyramid
        printf("%s%s  %s\n", spaces, hashes, hashes);
        strcpy(spaces, "");
        numS--;
        row++;
    }
    while (row <= height);
}

File: pset6/mario/more/mario.py
from cs50 import get_int


def main():
    while True:
        # height is the height of the pyramid 
        height = get_int("Height: ")
        if height > 0 and height < 9:
            break
        
    get_pyramid(height)
    
    
def get_pyramid(n):
    for i in range(n):
        j = n - i - 1
        print(" " * j + "#" * (i + 1) + "  " + "#" * (i + 1), end="\n")
       
        
main()